### ISSUE TITLE
Invoke-CMApplyDriverPackage_Legacy.ps1 3.0.9 and Windows 7 integration

### ISSUE THREAD

**Issue by dannz87** [MSEndpointMgr/ModernDriverManagement#44]
This morning i did a switch from 2.2.1 to 3.0.9 and the first machine build crashed on detection, because there were a Windows 7 package in the detection list. The $DriverPackageDetails.OSName variable was NULL and the Confirm-OSName function just crashed. Cancelling the whole detection and returning an EVIL "1" exit code.  I've rolled back to 2.2.1....

By looking at the code, i've realised that Windows 7 have been completely removed from the detection methods since version 3.0.0.

I know its the end of support for Windows 7, but many Governments are paying for the ESU and still managing it. Its sadly still "actual".

So i've looked at the code and changed the line 1059 : 
`if ($DriverPackageItem.PackageName -match "^.*Windows.*(?<OSName>(10)).*") {`
to
`if ($DriverPackageItem.PackageName -match "^.*Windows.*(?<OSName>(10|7)).*") {`

After that, the trouble was with the build detection, so i've added this under line 614
```
            "6.1.*" {
				$OSName = "Windows 7"
				switch (([System.Version]$InputObject).Build) {
					"7601" {
						$OSVersion =  "7601"
					}
                }					             
            }

```

I've havent put it in production yet, i'm asking if there is any other issues expected.

**Comment by vartaxe** [comment]
yes sadly i would also love to see windows 7 still supported and webservice in parallel to admin service would also be nice
